extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -31,146 +32,183 @@ if (context.RunTest) {
 				attempts--;
 				await utils.sleep(1000); // To ensure the providers are registered.
 			}
+			schemaCompareTester = new SchemaCompareTester();
 			console.log(`Start schema compare tests`);
 		});
 		test('Schema compare dacpac to dacpac comparison', async function () {
+			await schemaCompareTester.SchemaCompareDacpacToDacpac();
+		});
+		test('Schema compare database to database comparison and script generation', async function () {
+			await schemaCompareTester.SchemaCompareDatabaseToDatabase();
+		});
+		test('Schema compare dacpac to database comparison and script generation', async function () {
+			await schemaCompareTester.SchemaCompareDacpacToDatabase();
+		});
+	});
+}
+
+class SchemaCompareTester {
+	private static ParallelCount = 1;
+
+	@stressify({ dop: SchemaCompareTester.ParallelCount })
```

Comment:
suggestion: The SchemaCompareTests take a long time to run, so a future checkin,
we should pass a runtime: parameter to stressify to increase the amount of run time available to these tests and also see if they can be made runnable with dop > 1 to get more iterations in a shorter time.

Reply:
Will try and update if it works